lib/plugins/features/movies/index.js

```diff
@@ -1,22 +1,39 @@
 'use strict';
 
-const Controller     = require('./controller');
-const MovieValidator = require('../../../validators/movie');
+const Controller        = require('./controller');
+const MovieValidator    = require('../../../validators/movie');
+const MovieGetValidator = require('../../../validators/movieget');
```

Comment:
nit: i think we tend to kebab-case our file names (e.g. `movie-get`), camelCase our variables, and snake_case our JSON variables, SQL columns, and constants in our node codebases. In this case, you could kebab-case `movie-get` or optionally restructure the folders to look like `.../validators/movie/retrieve` and `.../validators/movie/create` like how we do in our codebase. We structure it in the latter manner b/c we have so many validators, but it's not necessarily required here.

If you do opt to just rename the file name to `movie-get`, make sure to alphabetize the constants here (so `MovieGetValidator` comes before `MovieValidator`).